examples/rmg/commented/input.py

```diff
@@ -223,6 +223,11 @@
     # maximumHeavyAtoms=20,
     # maximum radicals on a molecule
     maximumRadicalElectrons=1,
+    # maximum number of singlet carbenes (lone pair on a carbon atom) in a molecule
+    maximumSingletCarbenes=1,
+    # maximum number of radicals on a molecule with a singlet carbene
+    # should be lower than maximumRadicalElectrons in order to have an effect
```

Comment:
change "lower" to "lower or equal"

Reply:
If it's equal, then it still won't have any effect, since it will enforce the same limitation as `maximumRadicalElectrons`, which returns first.

Reply:
what I'm trying to say is that it's confusing to have comment written as "lower than" but you have example with "equal"

Reply:
I'll change the value to 0 then.